doc/api/process.markdown

```diff
@@ -495,6 +495,29 @@ An example of the possible output looks like:
 
 If `process.connected` is false, it is no longer possible to send messages.
 
+## process.cpuUsage()
+
+Returns the user and system cpu time usage of the current process, in an object
+with properties `user` and `system`, whose values are arrays of integers as
+`[seconds, nanoseconds]` tuples. These values measure time spent in user and
+system code respectively, and may end up being greater than actual elapsed time
+if multiple cpu cores are performing work for this process.
+
+You may pass in the result of a previous call to `process.cpuUsage()` to get a
```

Comment:
Sounds good.  Note, I copied off of `hrtime()` which has the same usage of 'You ...', because that's what I copied from.  Happy to do a drive-by fix to that as well ...